paperqa/agents/tools.py

```diff
@@ -412,10 +405,10 @@ async def complete(
 
         logger.info(
             f"Completing '{state.session.question}' as"
-            f" '{'a success' if has_successful_answer else 'unsure'}'."
+            f" '{'sure' if has_successful_answer else 'unsure'}'."
         )
         # Return answer and status to simplify postprocessing of tool response
-        return f"{'Success' if has_successful_answer else 'Unsure'} | {state.status}"
+        return f"{'Sure' if has_successful_answer else 'Unsure'} | {state.status}"
```

Comment:
Maybe use `Certain` instead? `Sure` feels weird here (but I get that it is symmetric with Unsure)

Reply:
Nice suggestion, implemented